tests/Command/ImportFileCommandTest.php

```diff
@@ -18,29 +21,115 @@ class ImportFileCommandTest extends TestCase {
      */
     private CommandTester $commandTester;
 
+    /**
+     * @var string
+     */
+    private string $filePath;
+    /**
+     * @var ProductService|MockObject
+     */
+    private $productService;
+    /**
+     * @var ServiceImportFactory|MockObject
+     */
+    private $importFactory;
+
+    /**
+     * @var string
+     */
+    private string $badFilePath;
+
     protected function setUp(): void {
         $this->projectDirectory = getcwd();
-        $productService = $this->getMockBuilder(ProductService::class)
+        $this->productService = $this->getMockBuilder(ProductService::class)
             ->disableOriginalConstructor()
             ->getMock();
 
-        $importFactory = $this->getMockBuilder(ServiceImportFactory::class)
+        $this->importFactory = $this->getMockBuilder(ServiceImportFactory::class)
             ->disableOriginalConstructor()
             ->getMock();
 
-        $importFactory->method('getImportService')
-            ->willReturn(new GeneralImportService($productService));
+        $this->filePath = $this->projectDirectory . '/tests/storage/stock.csv';
+
+        $file = fopen($this->filePath, 'wb+');
+        $data = [
+            ProductService::PRODUCT_HEADERS,
```

Comment:
Also I don't think you should use this constant there, just use the string values there